DEV_ENVIRONMENT.md

```diff
@@ -6,22 +6,21 @@
    - On Windows: Follow the [instructions](https://docs.bazel.build/versions/master/install-windows.html#5-optional-install-compilers-and-language-runtimes)
    to install [`MSYS2`](https://www.msys2.org/) and the listed "Common MSYS2 packages".
    Afterwards add `C:\msys64\usr\bin` to the `PATH` environment variable.
-3. Run `yarn global add gulp` to install gulp.
```

Comment:
I can't comment above, but can you also update the `Node` and `Yarn` requirements above? They are no longer up-to-date.

Reply:
I think I'd rather remove the versions since they'll go out of date eventually. Since we set the `engines` in the `package.json`, people will get a message if their version is wrong.